.github/ISSUE_TEMPLATE/content_issue_report.yml

```diff
@@ -0,0 +1,31 @@
+name: Content Issue Report
+description: Report a content issue on the website.
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking your time to report this content issue! If you need real-time help, join us on [Discord](https://discord.quiltmc.org).
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the issue
+      description: "Please describe what went wrong with the website content."
+    validations:
+      required: true
+
+  - type: textarea
```

Comment:
This field isn't necessary for content issues. Content issues are things like typos, inaccuracies, outdated information, etc.